docs/GettingStarted/DockerComposeSetup.md

```diff
@@ -33,8 +33,87 @@ sidebar_position: 1
 
 Support for database schema migration was introduced to DevLake in v0.10.0. From v0.10.0 onwards, users can upgrade their instance smoothly to a newer version. However, versions prior to v0.10.0 do not support upgrading to a newer version with a different database schema.
 
-<br/>
+<br/> 
 
+## FAQ
+### 1. Can I use an external Database service instead of running Database in Docker?
+   
+    Yes, you can simply comment out mysql/grafana part in docker-compose.yml and update some configurations in .env befor you run `docker compose up -d`, here are the steps:
```

Comment:
Text change:

Yes, you can simply comment out the 'mysql' part in 'docker-compose.yml' and update some configurations in '.env' before you run `docker compose up -d`, here are the steps:

Reply:
fixed